client/ext/bigquery/bigquery-module.js

```diff
@@ -0,0 +1,14 @@
+goog.provide('p3rf.perfkit.explorer.ext.bigquery.module');
+
+goog.require('p3rf.perfkit.explorer.ext.bigquery.BigqueryConfigModel');
+goog.require('p3rf.perfkit.explorer.ext.bigquery.BigqueryConfigService');
+goog.require('p3rf.perfkit.explorer.ext.bigquery.CurrentTimestampOptimizer');
+
+
+goog.scope(function() {
+  const bigquery = p3rf.perfkit.explorer.ext.bigquery;
+  
+  bigquery.module = angular.module('pkx.bigquery', []);
```

Comment:
This is actually the first time I've used an extensibility-themed module as a dependency.  It doesn't matter, but FWIW I'll look at moving 'explorer' to 'pkx.explorer' so that the module name has a common prefix.